zuul-core/src/main/java/com/netflix/netty/common/ByteBufUtil.java

```diff
@@ -32,7 +31,7 @@
 public class ByteBufUtil {
 
     private static final boolean isAdvancedLeakDetection =
-            Lists.newArrayList(PARANOID, ADVANCED).contains(ResourceLeakDetector.getLevel());
+            ResourceLeakDetector.getLevel() == PARANOID || ResourceLeakDetector.getLevel() == ADVANCED;
```

Comment:
Not against this, but _could_ use ordinal:

```suggestion
            ResourceLeakDetector.getLevel().ordinal() >= ResourceLeakDetector.Level.ADVANCED.ordinal();
```

Either or is fine 🗡️